### ISSUE TITLE
ONNX failed to initialize

### ISSUE THREAD

**Issue by bigwinboy** [lshqqytiger/stable-diffusion-webui-amdgpu-forge#45]
I reinstalled it today, I can enter the interface, but every time I start it prompts ONNX failed to initialize, is it normal?
![2024-10-16 173137](https://github.com/user-attachments/assets/e9ca9beb-5278-452a-9eb2-316ecb07b80c)

**Comment by zhucezy** [comment]
I met this too. You may just downgrade your optimum to 1.22.0

**Comment by bigwinboy** [comment]
> I met this too. You may just downgrade your optimum to 1.22.0

Thank you

**Comment by kappel79** [comment]
I downgraded still have the error

**Comment by bigwinboy** [comment]
> I downgraded still have the error

Delete the folder named venv, modify the txt document called requirements_versions, find the optimum in it, just change it to optimum==1.22.0, save the txt document, and restart SD.

**Comment by zhucezy** [comment]
> > I downgraded still have the error
> 
> Delete the folder named venv, modify the txt document called requirements_versions, find the optimum in it, just change it to optimum==1.22.0, save the txt document, and restart SD.
Yes. if don't modify "optimum>=1.23" in requirements_versions, even you downgrade it, it will be upgraded to 1.23 automatically.

**Comment by zhucezy** [comment]
> > I downgraded still have the error
> 
> Delete the folder named venv, modify the txt document called requirements_versions, find the optimum in it, just change it to optimum==1.22.0, save the txt document, and restart SD.
Yes. if don't modify "optimum>=1.23" in requirements_versions, even you downgrade it, it will be upgraded to 1.23 automatically.

**Comment by TheFerumn** [comment]
Today optimum was updated to 1.23.0. I haven't tested yet but yesterday it couldn't launch on 1.22.0 so it might be fixed if you will update to 1.23.0

**Comment by bigwinboy** [comment]
> Today optimum was updated to 1.23.0. I haven't tested yet but yesterday it couldn't launch on 1.22.0 so it might be fixed if you will update to 1.23.0

Deleted the venv folder, re-downloaded, and used 1.23.0 for optimum. The error code is still there, but it looks like this:
ONNX failed to initialize: module 'modules.devices' has no attribute 'backend'
